cime_config/cesm/machines/python.case.test

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python2
```

Comment:
This is a prototype for the case.test file in python.   It is not yet being used.